creator-node/test/lib/genericBullQueueMock.js

```diff
@@ -0,0 +1,42 @@
+const PrometheusRegistry = require('../../src/services/prometheusMonitoring/prometheusRegistry')
+const Bull = require('bull')
+
+const config = require('../../src/config')
+
+// Mock monitoring queue that sets monitor values on construction
+class GenericBullQueue {
+  constructor() {
+    this.queue = Bull('genericBullQueue', {
+      redis: {
+        host: config.get('redisHost'),
+        port: config.get('redisPort')
+      },
+      defaultJobOptions: {
+        removeOnComplete: 0,
+        removeOnFail: 0
+      }
+    })
+    const prometheusRegistry = new PrometheusRegistry()
+    prometheusRegistry.startQueueMetrics(this.queue)
+
+    this.queue.process(1, async (job, done) => {
```

Comment:
lets remove the `done` param and line 31 `done()` function call. if process takes an async fn, done does not need to be used

Reply:
Done